lib/taskana-core/src/test/java/acceptance/priorityservice/TestStaticValuePriorityServiceProvider.java

```diff
@@ -0,0 +1,14 @@
+package acceptance.priorityservice;
+
+import java.util.OptionalInt;
+
+import pro.taskana.spi.priority.api.PriorityServiceProvider;
+import pro.taskana.task.api.models.TaskSummary;
+
+public class TestStaticValuePriorityServiceProvider implements PriorityServiceProvider {
+
+  @Override
+  public OptionalInt calculatePriority(TaskSummary taskSummary) {
+    return OptionalInt.of(5);
+  }
+}
```

Comment:
Idea: Move this to a nested Class in UpdateManualPriorityWithSpiAccTest
This allows other devs to see a little faster what the configured SPI does on that test.
+ It's one file less to worry about